pillar/top.sls

```diff
@@ -236,9 +236,9 @@ base:
     - edx
     - edx.sandbox
     - edx.ansible_vars
-  'G@roles:sandbox and P@environment:mitxpro-qa':
-    - match: compound
     - edx.ansible_vars.theme
+  'G@edx_codename:koa':
+    - match: compound
```

Comment:
We should place this further down the list as the pillar merging is last one wins. If it is placed above other ansible vars pillars then it is likely to get overwritten.